README.md

```diff
@@ -6,6 +6,23 @@
 
 A C++ library with APIs in Python and R for efficient storage and retrieval of single-cell data using [TileDB Embedded][tiledb].
 
+Note this branch, `main-old`, implements the [original specification](https://github.com/single-cell-data/TileDB-SOMA/blob/main-old/spec/specification.md). Please also see the `main` branch which implements the [updated specfication](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).
+
+# Language bindings and installation instructions
+
+* [Python](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/python)
+* [R](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/r)
+* [API documentation](https://tiledb-inc-tiledb-soma.readthedocs-hosted.com/en/latest/)
```

Comment:
does not need to be resolved to land this PR, but should we consider a separate RTD domain/account for SOMA?   CC:  @ambrosejcarr @aaronwolen

Reply:
@bkmartinjr we are on that (internal discussions well in place)